source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -486,8 +487,7 @@ bool lldb_private::formatters::swift::NSContiguousString_SummaryProvider(
   if (guts_sp)
     return StringGuts_SummaryProvider(*guts_sp, stream, options);
 
-  static ConstString g_StringGutsType(
-      SwiftLanguageRuntime::GetCurrentMangledName("$Ss11_StringGutsVD"));
+  static ConstString g_StringGutsType(MANGLING_PREFIX_STR "s11_StringGutsVD");
```

Comment:
This looks dangerous. This should be code that tree-matches the demangle tree instead.

Reply:
I mean, sure, but it's worse than any other formatter in-tree? I don't think it's a problem specific to this very instance.